lib/net.js

```diff
@@ -46,6 +47,8 @@ const errnoException = util._errnoException;
 const exceptionWithHostPort = util._exceptionWithHostPort;
 const isLegalPort = internalNet.isLegalPort;
 
+const kTimeout = require('internal/timers').kTimeout;
```

Comment:
Nit: We can use object destructuring here so that both the definitions can be done in a single line.